app/assets/stylesheets/application.scss

```
@@ -32,3 +32,18 @@
 .alert-alert {
   @extend .alert-danger;
 }
+
+// ナビバーの配置調整
+header {
+  height: 57px;
+  background-color: #007bff;
+}
+
+ul {
+  font-size: 13px;
+}
+
+.navbar {
+  margin: 0 auto;
+  width: 960px;
```

このようなケースは `max-width` がよいですね。

```suggestion
  max-width: 960px;
```